classes/actions/ShoppingfeedOrderImportActions.php

```diff
@@ -1085,6 +1085,8 @@ public function recalculateOrderPrices()
                 'total_products_wt' => Tools::ps_round($orderPrices['total_products_tax_incl'], 2),
                 'total_shipping_tax_incl' => Tools::ps_round($orderPrices['total_shipping_tax_incl'], 2),
                 'total_shipping_tax_excl' => Tools::ps_round($orderPrices['total_shipping_tax_excl'], 4),
+                'total_discount_tax_excl' => 0,
```

Comment:
Can you update an existant unit test to check this ?

Reply:
In fact, it's already done
https://github.com/shoppingflux/module-prestashop/blob/1.9.1/202/tests/OrderImport/OrderImportSimpleTest.php#L100
It seems the problem arrives when the PS generates an order invoice before the module recalculates the prices (hook actionValidateOrder). It's not native PS behavior and that's why this problem is so untypical

Reply:
Thank for the precision.